packages/react/src/components/DataGrid/index.tsx

```diff
@@ -0,0 +1,241 @@
+// import { Button } from "components/Button";
+import React, { useMemo } from "react";
+import {
+  flexRender,
+  getCoreRowModel,
+  getSortedRowModel,
+  TableOptions,
+  useReactTable,
+} from "@tanstack/react-table";
+import { Pagination, PaginationProps } from "components/Pagination";
+import classNames from "classnames";
+import { useCunningham } from "components/Provider";
+import { Loader } from "components/Loader";
+import {
+  paginationToPaginationState,
+  sortingStateToSortModel,
+  sortModelToSortingState,
+  useHeadlessColumns,
+} from "components/DataGrid/utils";
+import { RowSelectionState } from "@tanstack/table-core/src/features/RowSelection";
```

Comment:
Can you move package import above local import please.
We should add an eslint rule to detect and fix that automatically. You can take a look at Richie eslint configuration.